packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx

```diff
@@ -2,12 +2,8 @@ import { Card, Colors, H5 } from '@blueprintjs/core';
 import styled, { createGlobalStyle } from 'styled-components';
 
 interface HeaderContainerProps {
-    isEditMode: boolean;
-    isHovering?: boolean;
-}
-
-interface TitleWrapperProps {
-    hasDescription: boolean;
+    $isEditMode: boolean;
```

Comment:
why did you use this $?

Reply:
thanks for the review @rephus!!

nothing major but if you have a prop with `$` prefix it does not pass that prop to the DOM node thus it does not log bunch of warnings
https://styled-components.com/docs/api#transient-props